lib/internal/worker.js

```diff
@@ -166,6 +167,13 @@ class Worker extends EventEmitter {
         options.env);
     }
 
+    if (isMainThread && !locksInitialized) {
+      locksInitialized = true;
+      // Make sure that locks are initialized before active
+      // multithreading starts.
+      require('worker_threads').locks.query();
```

Comment:
Tiny nit: This is kind of implicit - I would prefer it if this was more explicit (like `.locks.initialize` that calls query internally).

Reply:
Well … I can add another method, but I don’t think `.locks.initialize()` as a name would work since that’s public, and we could use a symbol, but at that point we’re probably introducing more complexity than the comment here provides?